crates/swc_plugin_runner/Cargo.toml

```diff
@@ -17,12 +17,13 @@ parking_lot = "0.11"
 serde = {version = "1.0.126", features = ["derive"]}
 serde_json = "1.0.64"
 swc_atoms = {version = "0.2.7", path = '../swc_atoms'}
-swc_common = {version = "0.16.0", path = "../swc_common", features = ["plugin-rt"]}
-swc_ecma_ast = {version = "0.62.0", path = "../swc_ecma_ast"}
+swc_common = {version = "0.16.0", path = "../swc_common", features = ["plugin-mode"]}
```

Comment:
This should be `plugin-rt`.

Reply:
Updated - currently afaik feature definition for rt === mode. maybe we can just consolidate later depends on usecases?

Reply:
No, it should not be consolidated. It was intentionally separate, because users otherwise may try using features only for the runtime (`@swc/core`)